### DISCUSSION TITLE
Salvar as imagens em wwwroot

### DISCUSSION THREAD

**Discussion by Romulo-Queiroz** [balta-io/forum#379] [Tira Dúvidas] [answered]
Até onde cheguei na parte de salvar as imagens em métodos estáticos, eu não consigo de fato salvar porque não há autorização. Vi em alguns lugares dizendo que precisa autorizar ISS, algo do tipo, realmente não consegui resolver esse problema. Alguém pode me ajudar a entender o motivo desse erro?
![image](https://github.com/balta-io/forum/assets/88904173/89be94b9-fe56-4b12-8218-3526850fb955)

**Comment by henriqueweege** [comment, marked as answer]
Oi @Romulo-Queiroz , tudo bem?
Teria como postar um print do teu código? Passei por algo parecido há algum tempo, talvez eu consiga te ajudar.

**Reply by Romulo-Queiroz** [reply]
![image](https://github.com/balta-io/forum/assets/88904173/dbab345c-75e0-4fa6-b860-91afc2781166)
Na linha 114 ocorre a exceção

**Reply by henriqueweege** [reply]
@Romulo-Queiroz, repare que a variável "fullPath" está recebendo o caminho até o diretório que você quer salvar a imagem, mas não até a imagem em si.
Me parece que essa variável deveria ser "basePath"+"imagesFolder"+"fileName". Poderia tentar isso? Caso não funcione da um toque aqui ;)

**Reply by Romulo-Queiroz** [reply]
@henriqueweege Muito obrigado, você está correto! Apliquei a lógica que você apresentou e realmente funcionou :) Up

**Reply by henriqueweege** [reply]
@Romulo-Queiroz que bom! fico feliz em ajudar!!
